SCClassLibrary/Common/Control/ServerStatus.sc

```diff
@@ -151,7 +109,10 @@ ServerStatusWatcher {
 				};
 				serverBooting = false;
 				server.changed(\serverRunning);
-			}, onComplete);
+			}, {
+				server.sync;
```

Comment:
why was this added?

Reply:
to make sure the server process finishes all pending tasks from Server.tree before running onComplete.